### ISSUE TITLE
Sidebar accordion heading text does not meet WCAG 2.X AA guidelines for contrast

### ISSUE THREAD

**Issue by JeffersonBledsoe** [plone/volto#6319]
Current foreground colour: #517776
Current background colour: #F3F4F5
Contrast as calculated by [WebAIM contrast checker](https://webaim.org/resources/contrastchecker/): 4.48:1

Current screenshot:

<img width="306" alt="Screenshot 2024-09-25 at 12 40 26" src="https://github.com/user-attachments/assets/f1908c68-8117-45bf-bceb-9f7170531ad3">

## Proposed solution

- Change the foreground colour to use the same colour as the labels.
  - Introduces some more consistency into the sidebar

### Screenshot

<img width="312" alt="" src="https://github.com/user-attachments/assets/6376d9d7-1041-44af-ad23-d3ef66367fc2">

## Other options

- Decrease brightness of background colour
  - Currently using the `darkWhite` colour. This could impact other areas. We don't want to introduce new colours to keep consistency
- Increase brightness of foreground colour
  - Same reasoning as above. Using `@teal` for the foreground colour.

**Comment by davisagli** [comment]
The proposal looks fine to me.
